employee/src/main/java/org/esteco/anniversary/employee/manager/EmployeeManager.java

```diff
@@ -1,16 +1,19 @@
-package org.esteco.anniversary.employee.manager;
-
-import org.esteco.anniversary.employee.Employee;
-import org.springframework.stereotype.Component;
-
-import java.util.List;
-
-@Component
-public interface EmployeeManager {
-
-    int save(Employee emp);
-
-    Employee get(int empId);
-
-    List<Employee> getAll();
-}
+package org.esteco.anniversary.employee.manager;
+
+import org.esteco.anniversary.employee.Employee;
+import org.springframework.stereotype.Component;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Component
+@Transactional
+public interface EmployeeManager {
+
```

Comment:
I would call it a  Data access object rather than calling a Manager. e.g EmployeeDAO and EmployeeDAOImpl

Reply:
Yes, Will do that.

Reply:
Will remove the above test in the next PR.